client/src/Router.re

```diff
@@ -108,10 +108,12 @@ module LinkUnsafe = {
         ?title
         href
         onClick=(
-          self.handle((event, _self) => {
-            event->ReactEvent.Mouse.preventDefault;
-            pushUnsafe(href);
-          })
+          self.handle((event, _self) =>
+            if (!event->ReactEvent.Mouse.ctrlKey) {
```

Comment:
You probably also want a condition for middle click on the mouse

Reply:
@kyldvs Thank you for reminding me.